server/errors/errors.go

```diff
@@ -116,6 +116,10 @@ func DuplicateTupleInWrite(tk *openfgapb.TupleKey) error {
 	return status.Error(codes.Code(openfgapb.ErrorCode_cannot_allow_duplicate_tuples_in_one_request), fmt.Sprintf("duplicate tuple in write: user: '%s', relation: '%s', object: '%s'", tk.GetUser(), tk.GetRelation(), tk.GetObject()))
 }
 
+func WriteToIndirectRelationError(reason string, tk *openfgapb.TupleKey) error {
+	return status.Error(codes.Code(openfgapb.ErrorCode_invalid_tuple), fmt.Sprintf("Invalid tuple '%s'. Reason: %s", tk.String(), reason))
```

Comment:
Considering the reason is always: `Attempting to write directly to an indirect only relationship`

Does it make sense to pass it in each case? Couldn't be added here instead?

Reply:
I thought about that, but I wasn't sure if we would/do want to have a unique message for each so I kept it as an input.  Completely open to suggestions here, though, I don't have a strong feeling one way or another